packages/storybook/stories/MeetingComposite/snippets/Server.snippet.tsx

```diff
@@ -0,0 +1,63 @@
+// Contoso server to create a new user and thread.
```

Comment:
Server name is misleading here.
Makes me think it is a REST API
Looks more like a util file to me.

Reply:
The idea behind this is that this is the code your server would use and would respond as a backend API (this comment is here because this snippet is shown directly in the docs)
![image](https://user-images.githubusercontent.com/2684369/126693292-af14e6e2-040c-4740-babb-e4c36e8953a5.png)

I'll keep this file consistent with the chat and calling composite docs (this was mostly copy paste from them) and we can do a seperate PR if this needs worded better